src/mixins/filing-template-mixin.ts

```diff
@@ -200,6 +200,15 @@ export default class FilingTemplateMixin extends DateMixin {
         filing.incorporationApplication.incorporationAgreement = {
           agreementType: this.getIncorporationAgreementStep.agreementType
         }
+
+        if (this.getEntityType === CorpTypeCd.BC_ULC_COMPANY) {
```

Comment:
I think there's a global/store getter for this already that you can use here, and on line 322.

Reply:
Done.